core/app/serializers/uffizzi_core/api/cli/v1/projects/deployments_serializer/user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UffizziCore::Api::Cli::V1::Projects::DeploymentsSerializer::UserSerializer < UffizziCore::BaseSerializer
+  type :user
+
+  attributes :email
```

Comment:
I think we should probably add the Github username and type (Github/Gitlab in the future) if it exists